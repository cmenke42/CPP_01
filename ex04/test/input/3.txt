Hello there are some "e" in this text.

ea

E

E

e
e
eeeee
