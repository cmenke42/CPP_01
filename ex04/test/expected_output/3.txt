H+llo th+r+ ar+ som+ "+" in this t+xt.

+a

E

E

+
+
+++++
